c_glib/test/test-table.rb

```diff
@@ -210,5 +210,21 @@ def test_concatenate
                      table.slice(-2, 2))
       end
     end
+
+    def test_combine_chunks
+      table = build_table(
+        "visible" => Arrow::ChunkedArray::new([build_boolean_array([true, false, true]),
+                                               build_boolean_array([false, true]),
+                                               build_boolean_array([false])])
+      )
+      combined_table = table.combine_chunks
+      assert_equal(table, combined_table)
+
+      chunked_array = combined_table.get_column_data(0)
+      assert_equal(1, chunked_array.n_chunks)
+
+      assert_equal([true, false, true, false, true, false],
+                   chunked_array.get_chunk(0).values)
```

Comment:
One assertion is better than multiple assertions for debug propose:

```ruby
combined_table = table.combine_chunks
all_values = combined_table.n_chunks.times.collect do |i|
  combined_table.get_column_data(i).values
end
assert_equal([[true, false, true, false, true, false]],
             all_values)
```

if `assert_equal(1, chunked_array.n_chunks)` is failed, we only know the number of wrong chunks. But the above one assertion shows us not only the number of chunks but also details of each chunk.

Reply:
I see. I'll rewrite it. Thanks.